bin/GameIndex.dbf

```diff
@@ -18573,6 +18800,16 @@ Region = NTSC-K
 Compat = 5
 OPHFLagHack = 1
 ---------------------------------------------
+Serial = SLKA-25372 
+Name   = Black 
+Region = NTSC-K 
+vuClampMode = 0 // Fix broken graphics ingame. 
+--------------------------------------------- 
+Serial = SLKA-25374
+Name   = Fifa Street 2
+Region = NTSC-K
+GIFFIFOHack = 1 // Fix flag corruptions
+---------------------------------------------
```

Comment:
Okay this is the merge conflict probably, can you exclude/remove the two added serials ?

Reply:
Serial = SLKA-25372 
Serial = SLKA-25374